src/resources/js/commerce/gateway/paypal/checkout.js

```diff
@@ -816,4 +846,4 @@ tribe.tickets.commerce.gateway.paypal.checkout = {};
 
 	$( obj.ready );
 
-} )( jQuery, tribe.tickets.commerce.gateway.paypal.checkout );
+} )( jQuery, tribe.tickets.commerce );
```

Comment:
I think this one should still be `tribe.tickets.commerce.gateway.paypal.checkout`. Any particular reason why we're updating this one?

Reply:
Same opinion. Keep `tribe.tickets.commerce.gateway.paypal.checkout` as `obj`, then just add `tribe.tickets.commerce.billing` as `billing`